README.md

```diff
@@ -22,3 +22,155 @@ e.g.
 ```
 npm i @fastify/cors -w @mykomap/back-end --save
 ```
+
+## Installation and updates
+
+Conceptually, installation of these applications require:
+- deploying the front-end as content to be served on the web
+  - configure it to use the correct path for the back end (typically `/api`)
+  - also configure API keys for GlitchTip and MapTiler services
+- deploying the back end to be run as a persistent node process
+  - configure it to use the right path to the data folder
+- deploy the data in the path expected by the back-end
+- reverse-proxying the back end to be accessible on the same domain as the front end
+
+### Deploying on a DCC server, manually
+
+DCC deployment uses a Linux server running Apache, an appropriate
+installation of NodeJS, with SystemD that supports user-mode
+services. A dedicated user is used to deploy the application, and this
+is exposed to Apache via configuration.
+
+This situation currently set up via some Ansible playbooks in the DCC
+`technology-and-infrastructure` project. The instructions following
+assume this context.
+
+For the descriptions below, we make the following assumptions for generality:
+
+ - `$SERVER` is the ssh URI used for the hostname being deployed to.
+ - `$USER` is the user being deployed to on that host.
+ - `$GIT_WORKING` is the directory where `mykomap-monolith` repository
+   is checked out (or if not checked out, unpacked - in which case
+   replace `git clone` or `git pull` with an appropriate unpacking
+   process)
+ - `$DATA_DIR` is the path to the directory containing data for the back-end.
+ - A file exists at `$DEPLOY_ENV` defining the environment variables
+   specifically needed for deployment.
+ 
+Examples, at the time of writing, of the typical case for these are:
+
+    SERVER=dev-2
+    USER=broccoli
+    GIT_WORKING=/home/$USER/gitworking/mykomap-monolith
+    DEPLOY_ENV=/home/$USER/gitworking/deploy.env
+    DATA_DIR=/home/$USER/deploy/data
+
+Likewise, an example of the contents of the file `$DEPLOY_ENV`, but
+with actual secret values redacted:
+
+    export USERDIR=/home/$USER
+    export DEPLOY_DEST=$USERDIR/deploy
+    export PROXY_PORT=1$UID
+    export PROXY_PATH=/api
+    export DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/$UID/bus
+    export GLITCHTIP_KEY=*REDACTED*
+    export MAPTILER_API_KEY=*REDACTED*
+
+*Note: paths here should be absolute - relative paths will not work in
+general.*
+
+*Note, the environment variables `USER` and `UID` should already be
+set by your shell by default.  `DBUS_SESSION_BUS_ADDRESS` should also
+be set in principle be if you are logged in as that user - but in
+practise is not. This is needed for the deploy script to run
+`systemctl` in user-mode.*
+   
+
+#### Install
+
+All these steps assume that `$DEPLOY_ENV` has been created and
+populated appropriately, as per the example above.
+
+##### With elevated privileges
```

Comment:
I think this could be a numbered step e.g. `Step 1: with elevated privileges`

since it wasn't clear on my initial scan of the doc that these were steps to perform in order, they looked like different info sections